src/api/java/baritone/api/pathing/goals/GoalYLevel.java

```diff
@@ -57,6 +57,10 @@ public static double calculate(int goalY, int currentY) {
         return 0;
     }
 
+    public int getLevel() {
```

Comment:
just make the level `public final`, dont use a getter. all the other settings have public final x y z etc